lib/transform/template.js

```diff
@@ -0,0 +1,64 @@
+const { indentLines } = require('../utils');
+
+const templateRecast = require('ember-template-recast');
+
+const b = templateRecast.builders;
+
+const PLACEHOLDER = '@@@PLACEHOLDER@@@';
+
+module.exports = function transformTemplate(
+  template,
+  tagName,
+  elementId,
+  classNames,
+  classNameBindings,
+  attributeBindings,
```

Comment:
can we change these to be "named" parameters instead?

see https://github.com/ryanmcdermott/clean-code-javascript#function-arguments-2-or-fewer-ideally 😉

Reply:
Good catch, done. I remember I wanted to clean this up, after making all tests to pass, but then apparently forgot...